smartcontract/contracts/RadaNFTToken.sol

```diff
@@ -2,21 +2,23 @@
 pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 import "@openzeppelin/contracts/utils/Context.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 
-contract RadaNFTToken is ERC721URIStorage, Ownable {
+contract RadaNFTToken is ERC1155, Ownable {
     using Counters for Counters.Counter;
     Counters.Counter private _tokenIds;
-
+    
     mapping(address => bool) private whitelist;
+    
+    mapping(uint => string) public tokenIdToURI;
 
-    constructor() ERC721("Mintable Token", "MINT") {}
+    constructor() ERC1155("https://example/api/item/{id}.json") {}
```

Comment:
add comment 'what does the contructor's param do?

Reply:
This is just for the first ERC1155 that needs a string to init when deploy. No worry about this. We do not use this later.